src/server/flows/CliFlow.ts

```diff
@@ -34,8 +35,9 @@ export class CliFlow implements IPluginMiddleware<any> {
       const token = await this.provider.getToken(code)
       const username = await this.provider.getUsername(token)
       const groups = await this.provider.getGroups(token)
+      const teams = await this.provider.getTeams(username, getConfig(this.provider.getConf(), "org"), token)
```

Comment:
The provider already has access to the config, so it doesn't need to be passed in. See some existing examples in  GitHubAuthProvider. Then we also don't need `Provider.getConf` any more.

Reply:
In `GitHubAuthProvider`, `config` is private if I'm not mistaken. Do you have an example how to access the config object without the `getConf` in `CliFlow` or `WebFlow` ? 

Or maybe you mean that I should create a `getOrg` function in `GithubAuthProvider` (and update the interface) instead of `getConf` function :

```javascript
  getOrg() {
    return getConfig(this.config, "org")
  }
```

Reply:
There is no reason to expose such a method because `getConfig` can be accessed by any file. So any file that needs access to configuration properties just imports `getConfig` and calls it directly. There is no need to pass it around or expose it.

Reply:
If in doubt, you could mimic how the org property is used since the team property is used in essentially the same way.

Reply:
`getConfig` needs a `Config` parameter (from [there](https://github.com/n4bb12/verdaccio-github-oauth-ui/blob/master/src/server/plugin/Config.ts#L30-L37)), do you an example I can use ?